--- ../vendor/laravel/framework/src/Illuminate/Foundation/Console/ServeCommand.php.orig	2024-12-17 23:04:30.193966016 +0100
+++ ../vendor/laravel/framework/src/Illuminate/Foundation/Console/ServeCommand.php	2024-12-17 23:05:35.926189704 +0100
@@ -377,13 +377,7 @@
      */
     public static function getRequestPortFromLine($line)
     {
-        preg_match('/http:\/\/.*:(\d+)/', $line, $matches);
-
-        if (! isset($matches[2])) {
-            throw new \InvalidArgumentException("Failed to extract the request port. Ensure the log line contains a valid port: {$line}");
-        }
-
-        return (int) $matches[2];
+        return 8000;
     }
 
     /**
